src/UI/Tables/Common.elm

```diff
@@ -3,6 +3,7 @@ module UI.Tables.Common exposing
     , ColumnWidth, columnWidthPortion, columnWidthPixels
     , Row, ToRow, rowEmpty, rowCellText, rowCellButton
     , Cell, cellFromText, cellFromButton
+    , cellFromCustom, rowCellCustom
```

Comment:
```suggestion
    , Row, ToRow, rowEmpty, rowCellText, rowCellButton, rowCellCustom
    , Cell, cellFromText, cellFromButton, cellFromCustom
```

This way we keep them grouped by context

Reply:
For the elm-format to leave it in this way, you must sort it to in the `@docs` and not here

Reply:
>For the elm-format to leave it in this way, you must sort it to in the @docs and not here

Thanks @PedroHLC for the tip, may I read some documentation how `@docs` work?